docs/shapes.rst

```diff
@@ -0,0 +1,352 @@
+    center: PixCoord(x=55.35205711214607, y=40.0958313892697)
+    radius: 36.93290808340659
+
+Also to convert a :class:`~regions.PixelRegion` to a
+:class:`~regions.SkyRegion`, call the :meth:`~regions.PixelRegion.to_sky` method:
+
+.. code-block:: python
+
+    >>> sky_reg = pix_reg.to_sky(wcs)
+    >>> print(sky_reg)
+    Region: CircleSkyRegion
+    center: <SkyCoord (ICRS): (ra, dec) in deg
+        (50., 10.)>
+    radius: 30.0 deg
+
+.. _sh-meta:
+
+Meta Data
+---------
+
+A :class:`~regions.Region` has ``meta`` and ``visual`` attributes which
+stores the meta data of the region. Since, this package supports various file
+formats it is necessary to handle the meta attributes supported by them.
+To handle them there are :class:`~regions.RegionMeta` and
+:class:`~regions.RegionVisual` for meta and visual attributes respectively.
+They are subclasses of the python dictionary (`~dict`).
+
+The meta attribute provides additional data about regions such as labels, tags,
+comments, name, etc.. which are used for non-display tasks.
+It also stores the spectral dimensions of the region.
+These classes , for now, just checks whether the key is valid or not.
+
+The valid keys for :class:`~regions.RegionMeta` class are:
+
+1. ``label``/``text``: CRTF, DS9 (text label for a region)
+
+    - Ex: meta['label'] = 'this is a circle'
+
+2. ``symbol``/``point``: CRTF, DS9 (Symbol for which a point region is described)
+
+    - Ex: meta['symbol'] = 'point marker'
+
+3. ``include``: CRTF, DS9 (Region inclusion)
+
+    - Possible Value: True, False
+
+    - Ex: meta['include'] = True
+
+4. ``frame``: CRTF (Frequency/Velocity Axis)
+
+    - Possible values: 'REST', 'LSRK', 'LSRD', 'BARY', 'GEO', 'TOPO', 'GALACTO', 'LGROUP', 'CMB'
+
+    - Default: image value
+
+    - Ex: meta['frame'] = 'TOPO'
+
+5. ``range``: CRTF (Frequency/Velocity Range)
+
+    - Possible units: GHz, MHz, kHz, km/s, Hz, channel, chan (=channel)
+
+    - Default: image range
+
+    - Format: [min, max]
+
+    - Ex: meta['range'] = [-320 * u.m/u.sec, -330 * u.m/u.s]
+
+6. ``veltype``: CRTF (Velocity Calculation)
+
+    - Possible values: 'RADIO', 'OPTICAL', 'Z', 'BETA', 'GAMMA'
+
+    - Default: image value
+
+    - Ex: meta['veltype'] = 'RADIO'
+
+7. ``restfreq``: CRTF (Rest Frequency)
+
+    - Possible values: `~astropy.units.Quantity` object
+
+    - Default: image value
+
+    - Ex: meta['restfreq'] = Quantity("1.42GHz")
+
+8. ``corr``: CRTF (Correlational Axis)
+
+    - Possible values: 'I', 'Q', 'U', 'V', 'RR', 'RL', 'LR', 'LL', 'XX', 'XY',
+      'YX', 'YY', 'RX', 'RY', 'LX', 'LY', 'XR', 'XL', 'YR', 'YL', 'PP', 'PQ',
+      'QP', 'QQ', 'RCircular', 'LCircular', 'Linear', 'Ptotal', 'Plinear',
+      'PFtotal', 'PFlinear', 'Pangle'
+
+    - Default: all planes present in image
+
+    - Ex: meta['corr'] = ['X', 'Y']
+
+9. ``comment``: DS9
+
+    - Ex: meta['comment'] = 'Any comment for the region'
+
+10. ``coord``: CRTF (the coordinate system that the region follows)
```

Comment:
ds9 has this as well, but it's specified differently.  Does ds9 store this in a different way than CRTF within `regions`?  I'm not sure if it's supposed to be metadata or just incorporated directly into the `Coordinate` objects' frames; I think the latter.

Reply:
I also think we should remove this. Let this doc be here for now, as a remainder to remove this meta key along this doc in a separate PR.